### ISSUE TITLE
feature request:  timer improvement (quick way to set + show countdown)

### ISSUE THREAD

**Issue by WiliTest** [White-Tiger/T-Clock#137]
I often need a timer, I don't use the T-clock one for 2 reasons (I use the chrome extension "timer"):
+ it's too long to set a simple timer
+ it doesn't show the countdown (at least, not  in a good way)

So just some idea in case it could be done:
the timer should pop up in a single click (by pressing "timer), entering the value in a second click and the countdown should show somewhere on the taskbar (best) or maybe in a tooltip, or in a tiny popup (the current watchlist windows is not really useful:
+ it requires 2 clicks (+ the click to set it)
+ it's too big (we only need to see the countdown, several countdown could be optional)
+ it should remains on the top
(optional: 3 buttons to pause, start, reset the timer)

Having the same things that the chrome extension "timer" when we click timer would be perfect.

In any case, thanks a lot for your awesome work! (It's already super!)

**Comment by White-Tiger** [comment]
well.. “it should remain on the top” is already supported... so nothing the Watch List is lacking...
And yeah.. there aren't directly buttons, but there's still the right-click menu + corresponding hotkeys. _(a `pause` feature is missing though)_

**Comment by WiliTest** [comment]
Thanks for your reply, I apologize if I'm wrong, but, no the watch list doesn't remain on the top. It comes on the top when it first appears,  but when I click on the chrome windows it goes behind (but in any case, I can't use it for the reasons mentioned above).

**Comment by White-Tiger** [comment]
indeed, I thought it had a single "on top" option... The one that does that, is the "Click through when inactive" but depending on needs, that'll probably do "too much". I should probably consider to add a simply on-top option as well...

**Comment by WiliTest** [comment]
Yes! I just discovered the "click through when inactive" (opening the watch list, right click/option). It does keep it on top. So that is already done ✔  . It would be great to always show this popup as it is when it's not active (this little window is not intrusive, it's very light, no big maximize windows sign and stuff), and you would only need to remove the "timer" column to only show the "remaining" time
